jiant/tasks/qa.py

```diff
@@ -157,13 +158,13 @@ def _make_instance(passage, question, answer, label, par_idx, qst_idx, ans_idx):
             d["qst_idx"] = MetadataField(qst_idx)
             d["ans_idx"] = MetadataField(ans_idx)
             d["idx"] = MetadataField(ans_idx)  # required by evaluate()
-            if is_using_bert:
-                inp = para + question[1:-1] + answer[1:]
+            if is_using_pytorch_transformers:
+                inp = boundary_token_fn(para + question, answer)
```

Comment:
I think this will likely work as-is, but I think most BERT-for-QA models adjust the input to `para + [SEP] + qst + ans + [SEP]`, which is what the code was previously doing.

Reply:
Fixed.